DependencyInjection/Configuration.php

```diff
@@ -69,6 +69,18 @@ public function getConfigTreeBuilder()
                         ->scalarNode('priority')->defaultValue(1)->end()
                         ->scalarNode('changefreq')->defaultValue(UrlConcrete::CHANGEFREQ_DAILY)->end()
                         ->scalarNode('lastmod')->defaultValue('now')->end()
+                        ->scalarNode('default_locale')
```

Comment:
I do not think that this should belongs to `defaults`, maybe a `canBeEnabled` distinct section

Reply:
I try to rewrite this part